pkg/archive/jar_test.go

```diff
@@ -528,3 +567,11 @@ func TestStableGitProperties(t *testing.T) {
 		})
 	}
 }
+
+func readFileAndHandleException(path string) []byte {
```

Comment:
for these cases, we can use must() which I believe is already defined in this package but, otherwise, should be accessible elsewhere in the project:

data := must(os.ReadFile(...))